js/components/main.js

```diff
@@ -577,6 +586,25 @@ class Main extends ImmutableComponent {
     }
   }
 
+  onMinimizeClick (e) {
+    const window = electron.remote.getCurrentWindow();
```

Comment:
These getCurrentWindow calls are bad for perf in general, it's better to have the current window cached in the component via using 
`require('../app/renderer/currentWindow')`


See windowActions.js for an example